Cargo.toml

```diff
@@ -1,72 +1,7 @@
-[package]
-name = "fastcrypto"
-version = "2.0.0"
-license = "Apache-2.0"
-authors = ["Mysten Labs <build@mystenlabs.com>"]
-edition = "2021"
-publish = ["crates-io"]
-description = "Common cryptographic library used at Mysten Labs"
-repository = "https://github.com/MystenLabs/fastcrypto"
+[workspace]
+resolver = "2"
+members = [
+    "fastcrypto",
```

Comment:
so in the future all individual crates (fastcrypto-signature, fastcrypto-hash, fastcrypto-bulletproofs etc) will be extra members in the package?

Reply:
That is how rust [workspaces](https://doc.rust-lang.org/cargo/reference/workspaces.html) work.

Reply:
yeah my question is if our intention in the future is to retire `fastcrypto` and split it to individual crates that will live in a flat list along with `fastcrypto-derive`